dowhy/causal_refuters/linear_sensitivity_analyzer.py

```diff
@@ -326,9 +326,9 @@ def plot_estimate(self, r2tu_w, r2yu_tw):
         critical_estimate = self.null_hypothesis_effect
         contour_values = np.zeros((len(r2yu_tw), len(r2tu_w)))
         for i in range(len(r2yu_tw)):
-            y = r2tu_w[i]
+            y = r2yu_tw[i]

```

Comment:
Why were r2tu_w and r2yu_tw swapped in the inner/outer loops here?  It looks like this was a bug in the prior code?

Reply:
yes, this is to fix a bug in the prior code.